### ISSUE TITLE
Translations for 11.6.5 (localisation / locale / crowdin)

### ISSUE THREAD

**Issue by aubergine10** [CitiesSkylinesMods/TMPE#1344]
Note, this is just a planned list - don't add these to Crowdin yet (items listed here are subject to change, even after PRs are merged).

- [x] PR #1455
    * Some new UI groups - eg. highways
- [x] PR #1454
    * Hint text for overground/underground view https://github.com/CitiesSkylinesMods/TMPE/pull/1454#issuecomment-1059610762
- [x] PR #1404
    * Mod option on Overlays tab
- [x] PR #1301
    * Checkbox not in scope tooltip
- [x] PR #1343
    * In-game message about despawn due to pathfinder update
- [x] PR #1347
    * In-game message about despawned vehicles
- [x] PR #1341
    * Despawn group + filters on Maintenance tab
- [x] PR #1284
    * https://github.com/CitiesSkylinesMods/TMPE/issues/1288
    * Mostly done - remaining stuff moved to #1567
- [x] Issue #1268
    * `Overlay Opacity`

**Comment by aubergine10** [comment]
Added first batch of locale strings (mostly mod option stuff) to Crowdin for 11.6.5.3 milestone.

Some of them I changed the keys a bit - will send PR to reflect those changes in the mod options code.

I've not yet added the "allow despawn when despawn disabled" stuff as I think we need to significantly rethink the wording in that feature to make it much less confusing.

I'll add additional locale keys over the next week or so.
